app/services/ChargeCService.scala

```diff
@@ -54,22 +55,24 @@ object ChargeCService {
             case (Some(details), Some(chargeDetails)) =>
 
               val (name, isDeleted) = details
-              Employer(
+              Seq(Employer(
                 index,
                 name,
                 chargeDetails.amountTaxDue,
                 viewUrl(index, srn).url,
                 removeUrl(index, srn).url,
```

Comment:
Actually the above two comments don't apply to these specific changes on this branch so I will merge anyway.